aip/general/0236.md

```diff
@@ -141,9 +146,9 @@ message PolicyPreviewMetadata {
   absent.
   - It is present on the experiment once the "preview" method is used.
 - All `preview_metadata` fields **must** be output only.
-- `state` changes to and from `ACTIVE` and `INACTIVE` when the experiment is
-  started or stopped, which can only be done by the "preview" and "stop" custom
-  methods.
+- `state` changes between `ACTIVE` and `INACTIVE` when the experiment is
+  started and stopped being previewed. This happens when the "preview" and
```

Comment:
"This happens when the "preview" or "stopPreview custom methods are invoked, respectively."